### PR TITLE
Add script for updating Reaper packages

### LINKED ISSUE #19: Create a script for packaging the most recent version of Reaper
It may come in handy to have a shell script that will create a nix module for the most recent Reaper version. Similar to the `update.sh` in the Brave browser in nixpkgs.

### PR DESCRIPTION
**Pull request by andrewvious**
Closes #19 

Adds `update_reaper_flake_input.sh` which updates the Reaper packages to the **most** recent version and runs a check in CI that opens a pull request that will update the Reaper input to the flake.

### CODE REVIEW

**Comment by andrewvious** [comment]
Ready for review @eureka-cpu

File: .github/workflows/generate_reaper_plugin_functions.yml [hunk 1]
```diff
@@ -35,14 +38,36 @@ jobs:
...
           else
             echo "No update needed"
-            echo "::set-output name=update_needed::false"
+            echo "::set-output name=sdk_update_needed::false"
           fi
+          
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Check if rs update is needed
```
Comment (eureka-cpu) [inline .github/workflows/generate_reaper_plugin_functions.yml:50]: this name could be more descriptive

File: lib/update.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{ pkgs, ... }:
```
Comment (eureka-cpu) [inline lib/update.nix:1]: Not strictly necessary, but you could replace `pkgs` with the specific things you need from it (have a look at nix-watch.nix for an example)

**Review by eureka-cpu — CHANGES_REQUESTED** [review]
Looking good, nice work!

**Review by eureka-cpu — CHANGES_REQUESTED** [review]
Just need to change `lib/update.nix` to `scripts/update-reaper-flake-input.nix`

**Review by eureka-cpu — APPROVED** [review]

**Pull request merged**